spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -628,6 +628,17 @@ function recvPacket(
     // do sequence check before any state changes
     if channel.order == ORDERED || channel.order == ORDERED_ALLOW_TIMEOUT {
         nextSequenceRecv = provableStore.get(nextSequenceRecvPath(packet.destPort, packet.destChannel))
+        if (packet.sequence < nextSequenceRecv) {
+          emitLogEntry("recvPacket", {
+            sequence: packet.sequence, 
+            timeoutHeight: packet.timeoutHeight, 
+            port: packet.destPort, 
+            channel: packet.destChannel,
+            timeoutTimestamp: packet.timeoutTimestamp, 
+            data: packet.data
+          })
+        }
+
```

Comment:
Maybe add a comment indicating that the emission happens even if the transaction is aborted.